### PR TITLE
WIP: dev-java/openjdk and dev-java/openjfx: Latest jdk and openjfx release jdk15

### PR DESCRIPTION
**Pull request by dblaber**
Adding ebuild for the latest non-lts jdk release, openjdk15 and openjfx15.

Signed-off-by: Darren Blaber dmbtech@gmail.com

### CODE REVIEW

**Review by gyakovlev — APPROVED** [review]
looks good, you can leave ~ppc64 keyword, I tested it.

File: dev-java/openjdk/openjdk-15.0.0_p36.ebuild [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+	default
+	chmod +x configure || die
+}
+
+src_configure() {
+	# Work around stack alignment issue, bug #647954. in case we ever have x86
+	use x86 && append-flags -mincoming-stack-boundary=2
+
+	# Work around -fno-common ( GCC10 default ), bug #713180
+	append-flags -fcommon
```
[the remark this answers is not in the document]
Comment (ShadowsFriend) [reply dev-java/openjdk/openjdk-15.0.0_p36.ebuild:150]: I just tested this with GCC 10.2.0-r2 and the ebuild emerged fine without the workaround.

**Comment by jospezial** [comment]
surprise 17!
https://mail.openjdk.java.net/pipermail/jdk-dev/2021-September/006037.html

**Comment by ConiKost** [comment]
Can we close this? Openjdk-17 landed in tree :-)

**Comment by petteyg** [comment]
This is very stale.

**Comment by ConiKost** [comment]
Due staled, I will close here. Feel free to reopen for news about openjfx :-)

**Pull request closed without merging**